ztag/schema.py

```diff
@@ -106,6 +106,31 @@ class CensysString(WhitespaceAnalyzedString):
     "timestamp":DateTime(),
 })
 
+ztag_dsa_params = SubRecord({
+    "p":IndexedBinary(),
+    "q":IndexedBinary(),
+    "g":IndexedBinary(),
+    "y":IndexedBinary(),
+})
+
+ztag_ecdsa_public_key = SubRecord({
```

Comment:
Is this specific to SSH? There's a similar structure for ECDSA certificates. Do all of these fields actually need to be indexed?